modules/trusted-firmware-m/Kconfig.mbedtls_minimal.defconfig

```diff
@@ -47,3 +47,192 @@ config MBEDTLS_POLY1305_C
 
 config MBEDTLS_PSA_CRYPTO_STORAGE_C
 	default n
+
+
+config MBEDTLS_LEGACY_CRYPTO_C
+	default n
+
+
+config PSA_WANT_ALG_HMAC_DRBG
+	bool
+	default y
+
+config PSA_WANT_ALG_SHA_224
+	bool
+	default n
+
+config PSA_WANT_ALG_SHA_256
+	bool
+	default n
+
+
+
+config MBEDTLS
+	bool
+	default n
+
+config PSA_WANT_KEY_TYPE_DERIVE
+	bool
+	default n
+config PSA_WANT_KEY_TYPE_HMAC
```

Comment:
maybe move some off the saved newlines down here :wink:

Reply:
I have made an equal share redistribution of the newlines.